styles.css

```diff
@@ -0,0 +1,558 @@
+  flex-grow: 0;
+  margin-top: 0;
+  margin-bottom: 12px;
+}
+
+.image-icons {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px 151px 120px 24px;
+  width: 200px;
+  height: 40px;
+  flex: none;
+  order: 3;
+  gap: 18px;
+}
+
+/* Work Section */
+#resent-works {
+  font-family: "Crete Round", serif;
+}
+
+.heading-and-seperator {
+  align-items: center;
+  padding: 120px 24px 80px 24px;
+}
+
+.heading-and-seperator h1 {
+  margin: 0;
+  width: 327px;
+}
+
+.heading-and-seperator hr {
+  margin: 0;
+  width: 327px;
+}
+
+.heading {
+  margin-top: 120px;
+  text-align: center;
+  color: #1c1a19;
+  font-size: 40px;
+  font-weight: 400;
+  line-height: 52px;
+  height: 83px;
+}
+
+.horizontal-seperator {
+  border: 1px solid #212121;
+  margin: 24px 24px 80px;
+}
+
+.work-card-container {
+  display: grid;
+  grid-template-columns: auto;
+  justify-content: center;
+  gap: 24px 0;
+  width: 327px;
+  margin: 0 24px 120px 24px;
+}
+
+.work-card-container > div {
+  width: 327px;
+  margin: 0;
+}
+
+.img {
+  background-size: 100%;
+  width: 327px;
+  height: 237px;
+  background-repeat: no-repeat;
+}
+
+.remaining-img {
+  background-size: 100%;
+  width: 327px;
+  height: 386px;
+  background-repeat: no-repeat;
+}
+
+.img-one {
+  background-image: url("./images/portfolio/card-one.svg");
+}
+
+.img-two {
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("./images/portfolio/card-two.svg");
+}
+
+.img-three {
+  background-image: url("./images/portfolio/card-three.svg");
+}
+
+.img-four {
+  background-image: url("./images/portfolio/card-four.svg");
+}
+
+.img-five {
+  background-image: url("./images/portfolio/card-five.svg");
+}
+
+.img-six {
+  background-image: url("./images/portfolio/card-six.svg");
+}
+
+.img-seven {
+  background-image: url("./images/portfolio/card-seven.svg");
+}
+
+.card-hedaing {
+  width: 327px;
+  height: 36px;
+  margin: 24px 24px 12px 24px;
+  text-align: center;
+  color: #091e42;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  font-family: 'Crete Round', serif;
+}
+
+.card-heading + p {
+  width: 327px;
+  height: 147px;
+  color: #091e42;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  font-family: 'Inter', serif;
+  margin-top: 12px 12px;
+}
+
+.skills {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 327px;
+}
+
+.skills input[type=button] {
+  background: linear-gradient(0deg, #fff, #fff), linear-gradient(0deg, #8993a4, #8993a4);
+  padding: 10px 12px 10px 12px;
+  color: #091e42;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 20px;
+  font-family: 'Inter', serif;
+}
+
+.projects {
+  height: 48px;
+  width: 124px;
+  left: 0;
+  top: 259px;
+  border-radius: 0;
+  padding: 12px;
+  background: #ff6b00;
+  border: none;
+  color: #fff;
+  font-size: 17px;
+  font-weight: bold;
+  line-height: 24px;
+  font-family: 'Inter', serif;
+}
```

Comment:
Kindly consider checking the project/work section and checking to make sure that all features appear correctly before applying for the next review. It looks like your recent update has caused some of the project cards dissappearing as depicted below 👇. Currently, only the button of some of the projects are showing. Kindly check and fix that. 

![Screenshot from 2023-03-24 22-28-42](https://user-images.githubusercontent.com/65233860/227655324-dfb24c94-0358-4fc9-aa21-50ccd4f48247.png)


![Screenshot from 2023-03-24 22-28-51](https://user-images.githubusercontent.com/65233860/227655332-c8576830-4775-4dcf-9afa-0462f69a57a5.png)


![Screenshot from 2023-03-24 22-29-00](https://user-images.githubusercontent.com/65233860/227655342-90e6def9-ff03-4567-84fb-0f85ab639ba8.png)

Reply:
![image](https://user-images.githubusercontent.com/68891198/227656448-4ff4cf31-cb71-4b9c-9acf-8ed78453c511.png)
![image](https://user-images.githubusercontent.com/68891198/227656538-d330c049-65f8-4f90-b222-26a4fbcd3e26.png)

Reply:
It works from my side may I ask what do you think will be the problem [jerryowusu]?

Reply:
I think it is because in the style.css file, you stated a height of 48px, which is too small for the whole thing to appear. Kindly provide the correct height.

Reply:
Hi [jerryowusu] okay thank you, but I am using this style for the "See project" button, its height is correct as specified on Figma. Anyways thanks!